File: Arrays/Get_no_appears_ones/brute_force.py
def getSingleElement(arr):

    n = len(arr)

    for i in range(n):
        cnt = 0
        num = arr[i]
        for j in range(n):
            if num == arr[j]:
                cnt += 1
        if cnt == 1:
            return num
    return -1

if __name__ == "__main__":
    arr = [4,1,2,1,2]
    ans = getSingleElement(arr)
    print("The single element is:", ans)

File: Arrays/Get_no_appears_ones/brute_force.cpp
#include <bits/stdc++.h>
using namespace std;

int getSingleElement(vector<int>& arr){
    int n =  arr.size();
    
    for(int i=0; i<n; i++){
        int num = arr[i];
        int cnt =0;
        for(int j=0; j<n; j++){
            if (num == arr[j]){
                cnt ++;
            }
                
        }
        if (cnt == 1){
            return num;
        }
    }

}

int main(){
    vector<int> arr = {4, 1, 2, 1, 2};
    int ans = getSingleElement(arr);
    cout << "The single element is: " << ans << endl;

    return 0;

}